src/utils/__test__/highlights.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { getPercentFromDuration } from '../highlights';
+
+describe('getPercentFromDuration', () => {
+	it('percentage with 0 values', () => {
+		expect(getPercentFromDuration(0, 0)).toBe(0);
+	});
+	it('percentage for 0 mediaDuration', () => {
+		expect(getPercentFromDuration(10, 0)).toBe(0);
+	});
+	it('percentage from 100 media duration at 10t', () => {
+		expect(getPercentFromDuration(10, 100)).toBe(10);
+	});
+	it('percentage for a 50 barDivider', () => {
```

Comment:
What does this mean?

Reply:
The naming do not look so good, but generally it means this:
 we have 10 second from an 100-second duration and we need to represent it how much it would be in percents in an interval [0, barDivider=50]

Reply:
Maybe something like: `returns absolute value if max range is provided`...?